### PR TITLE
Remove transform on SVG element, which was causing problems in FF

### LINKED ISSUE #18: SVG transforms causing issues due to partial support
Firefox is not displaying the bundle visualisations at the correct size due to the transform="scale(scale(%f))" on the SVG element. This attribute is not supported in Chrome but is supported in FF [1] therefore scaling the bundle visualisation.

The SVG are displayed correctly when removing this attribute.

[1] http://caniuse.com/#feat=svg-html

### PR DESCRIPTION
**Pull request by makyo**
Fixes #18

### CODE REVIEW

File: canvas.go [hunk 1]
```diff
@@ -291,15 +291,13 @@ func (c *Canvas) Marshal(w io.Writer) {
 	// is to wrap the writer in a custom writer that panics
 	// on error, and catch the panic here.
 	width, height := c.layout()
-	scale := c.computeScale(width, height)
```
Comment (bac) [inline canvas.go]: Is computeScale still useful?

**Comment by bac** [comment]
Code :+1:.  Will QA later if no one gets to it.

**Comment by bac** [comment]
QA instructions would be helpful.

**Comment by makyo** [comment]
# QA

* Download https://api.jujucharms.com/charmstore/v4/bundle/rails-example-scalable-6/archive/bundle.yaml
* Run `go run examples/generatesvg.go rails.yaml > bundle.svg`
* Open that in Firefox, and compare it to https://api.jujucharms.com/charmstore/v4/bundle/rails-example-scalable-6/diagram.svg - it should no longer be scaled.  They should appear identical in Chrome

File: canvas.go (hunk 1 above)
Comment (makyo) [inline canvas.go]: Oh, I thought it was still used elsewhere.  Guess not!  Will remove.

**Comment by frankban** [comment]
:+1: and QA ok

**Comment by makyo** [comment]
Thanks, all.

**Pull request merged**